modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,216 @@
+* Updating DNS values.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+Consider the following use cases for creating manifest object that includes a customized `br-ex` bridge:
+
+* You want to make postinstallation changes to the bridge, such as changing the Open vSwitch (OVS) or OVN-Kubernetes `br-ex` bridge network. The `configure-ovs.sh` shell script does not support making postinstallation changes to the bridge.
+* You want to deploy the bridge on a different interface than the interface available on a host or server IP address.
+* You want to make advanced configurations to the bridge that are not possible with the `configure-ovs.sh` shell script. Using the script for these configurations might result in the bridge failing to connect multiple network interfaces and facilitating data forwarding between the interfaces.
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+. Create a NMState resource that contains the decoded base64 information for your customized `br-ex` bridge network. 
++
+.Example of an NMState resource YAML file with a configuration for a customized `br-ex` bridge network
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+# ...
+    interfaces:
+    - name: enp2s0 <1>
+      type: ethernet <2>
+      state: up <3>
+      ipv4:
+        enabled: false <4>
+      ipv6:
+        enabled: false
+    - name: br-ex
+      type: ovs-bridge
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      ipv6:
+        enabled: false
+        dhcp: false
+      bridge:
+        port:
+        - name: enp2s0 <5>
+        - name: br-ex
+      - name: br-ex
+      type: ovs-interface
+      state: up
+      copy-mac-from: enp2s0
+      ipv4:
+        enabled: true
+        dhcp: true
+      ipv6:
+        enabled: false
+        dhcp: false
+# ...
+----
+<1> Name of the interface.
+<2> The type of interface. This example creates a bridge.
+<3> The requested state for the interface after creation.
+<4> Disables IPv4 and IPv6 in this example.
+<5> The node NIC to which the bridge attaches.
+
+. Use the `cat` command to base64-encode the contents of the NMState resource YAML file:
++
+[source,terminal]
+----
+$ cat <base64_decoded_nmstate_configuration>.yaml | base64 <1>
```

Comment:
Could probably just call this <nmstate_configuration>.yaml since the fact that we're encoding it in this step should make it clear that this is the decoded content.

Reply:
Good point. Updated.